_styleguide.html

```diff
@@ -1108,8 +1109,8 @@
 <footer class="footer">
 </footer>
 
-    <script src="static/dist/bower_components/jquery/dist/jquery.min.js"></script>
-    <!--[if lt IE 9 ]> 
+    <script src="static/dist/js/vendor.min.js"></script>
```

Comment:
Only if left unconfigured - the idea is to bundle all of the plugins that you *know* will be required on every page, such as jQuery, Angular, etc. Everything else you could handle separately.